content/en/core/overview/self-hosting-multiple-nodes.md

```diff
@@ -0,0 +1,285 @@
+
+1. Download the clustered cht-couchdb docker-compose file.
+
+`curl -s -o ./cht/cht-couchdb-clustered.couchdb1.yml https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:4.0.1/docker-compose/cht-couchdb-clustered.yml`
+
+2. Create a `.env` file and populate it with the same content above as node 1.
+
+### Docker-compose modifications
+
+1. Since we want to run only one couchdb node per instance, delete couchdb.2 and couchdb.3 containers from the yml.
+2. Add this property under the `image` section: `container_name: couchdb.1`
+3. Go to the networks section and add the following:
+
+```
+  cht-overlay:
+    driver: overlay
+    external: true
+```
+
+4. Go to each of the containers and add the following:
+
+```
+    networks:
+      - cht-overlay
+```
+
+## Configuration - Node 3 - couchdb.2
+
+### Setup Docker
+
+1. `sudo apt update`
+2. `sudo apt install docker.io`
+3. `sudo apt install docker-compose`
+
+
+Create a `cht` directory under your home directory and download files on there.
+
+1. Download the clustered cht-couchdb docker-compose file.
+
+`curl -s -o ./cht/cht-couchdb-clustered.couchdb2.yml https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:4.0.1/docker-compose/cht-couchdb-clustered.yml`
+
+2. Create a `.env` file and populate it with the same content above as node 1.
+
+### Docker-compose modifications
+
+1. Since we want to run only one couchdb node per instance, delete couchdb.1 and couchdb.3 containers from the yml.
+2. Add this property under the `image` section: `container_name: couchdb.2`
+3. Go to the networks section and add the following:
+
+```
+  cht-overlay:
+    driver: overlay
+    external: true
+```
+
+4. Go to each of the containers and add the following:
+
+```
+    networks:
+      - cht-overlay
+```
+
+## Configuration - Node 4 - couchdb.3
+
+### Setup Docker
+
+1. `sudo apt update`
+2. `sudo apt install docker.io`
+3. `sudo apt install docker-compose`
+
+Create a `cht` directory under your home directory and download files on there.
+
+1. Download the clustered cht-couchdb docker-compose file.
+
+`curl -s -o ./cht/cht-couchdb-clustered.couchdb3.yml https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:4.0.1/docker-compose/cht-couchdb-clustered.yml`
+
+2. Create a `.env` file and populate it with the same content above as node 1.
+
+### Docker-compose modifications
+
+1. Since we want to run only one couchdb node per instance, delete couchdb.1 and couchdb.2 containers from the yml.
+2. Add this property under the `image` section: `container_name: couchdb.3`
+3. Go to the networks section and add the following:
+
+```
+  cht-overlay:
+    driver: overlay
+    external: true
+```
+
+4. Go to each of the containers and add the following:
+
+```
+    networks:
+      - cht-overlay
+```
+
+## Launch containers in the different nodes
+
+Please start the containers in the same order as they have been specified here.
+
+### Launch `couchdb.2`
```

Comment:
I think these names are changing to be FQDN. To save us updating the docs let's just say..

```suggestion
### Launch CouchDB node 2
```